api/rest/bundle_handler.go

```diff
@@ -211,7 +214,8 @@ func collectAll(ctx context.Context, done chan<- []string, dataFile io.WriteClos
 	done <- errors
 }
 
-func collect(ctx context.Context, c collector.Collector, zipWriter *zip.Writer) error {
+func collect(ctx context.Context, c collector.Collector, zipWriter *zip.Writer, timeout time.Duration) error {
+	ctx, _ = context.WithTimeout(ctx, timeout) //nolint: govet
```

Comment:
Why don't we just pass a different context as first parameter instead of shadowing the `ctx` parameter here and not using it at all?

Reply:
Fixed